WordPressAuthenticator.xcodeproj/project.pbxproj

```diff
@@ -1144,7 +1141,6 @@
 				"${BUILT_PRODUCTS_DIR}/WordPressKit/WordPressKit.framework",
 				"${BUILT_PRODUCTS_DIR}/WordPressShared/WordPressShared.framework",
 				"${BUILT_PRODUCTS_DIR}/WordPressUI/WordPressUI.framework",
-				"${BUILT_PRODUCTS_DIR}/lottie-ios/Lottie.framework",
```

Comment:
Worth updating the `pbxproj`'s project format version to a more recent one so that it supports using `xcfilelist` (making CocoaPods then use them and have those changes be outside of the `pbxproj`)?

Maybe for a separate PR though (similar to the idea you floated around about removing more files later… though the change in pbxproj format would not be a breaking one, so could be worth doing befor a new major version bump…? 🤷 )

Reply:
Oh good point. Thanks. I'll do that in a dedicated PR, to isolated that project structure change.